### ISSUE TITLE
Heel Bite damage modifier seems to be bugged in game

### ISSUE THREAD

**Issue by PositiveKoodoo** [Gaboros/Venture-Plan-Missions#2]
Right now the damage dealt modifier for the spell "Heel Bite" is certainly bugged, since it showed me a modification of 1.
So far I simply took this number but if I had to guess this will probably be fixed in a hotfix or patch by Blizzard.

https://github.com/Gaboros/Venture-Plan-Missions/blob/b522d4bdaa179892b3d02367ba69d214f8b7a608/vs-spells.lua#L372

**Comment by MMOSimca** [comment]
Yes; it's bugged, but even further than just that. It seems that modifier is actually positive, which means that instead of reducing, as the tooltip says, it actually increases the damage dealt.
